src/commands/dev/index.js

```diff
@@ -172,7 +175,25 @@ async function startProxy(settings, addonUrls, configPath, projectDir) {
     configPath,
   })
 
+  await mkdirp(path.resolve(projectDir, '.netlify'))
```

Comment:
`mkdirp` requires `util.promisify()` to be used with `await` I believe.

Reply:
The latest version has supported for promises by default

Reply:
Oh right, I missed that you also upgraded `mkdirp` :+1: